mimic/model/nova_objects.py

```diff
@@ -818,7 +819,8 @@ def request_delete(self, http_delete_request, server_id):
         server.update_status(u"DELETED")
         return b''
 
-    def request_action(self, http_action_request, server_id, absolutize_url):
+    def request_action(self, http_action_request, server_id, absolutize_url,
+                       regional_image_collection, image_store):
```

Comment:
`request_action` is a method on `RegionalServerCollection`.  `RegionalServerCollection` already has an attribute called `image_store`.  Why does this method also take a `regional_image_collection` and `image_store` rather than these being attributes on `RegionalServerCollection`?